src/pages/workspaces/workspace/Dashboard.test.js

```diff
@@ -100,17 +99,6 @@ describe('WorkspaceNotifications', () => {
   });
 });
 
-const defaultGoogleBucketOptions = {
-  googleBucketLocation: defaultLocation,
-  googleBucketType: locationTypes.default,
-  fetchedGoogleBucketLocation: undefined,
-};
-const defaultAzureStorageOptions = {
-  azureContainerRegion: undefined,
-  azureContainerUrl: undefined,
-  azureContainerSasUrl: undefined,
-};
-
 describe('BucketLocation', () => {
```

Comment:
All of these tests are Google-specific. Thus all of the Azure-related storage values should ALWAYS be undefined… so it is not correct to replace them with the new `defaultAzureStorageOptions`.

I would prefer to the have the workspaces tests restored back to how they were before. Those "default" values are meant to represent the initial state of a workspace and/or the values for cloud-specific storage when the workspace being rendered is of the "other" type. If we are rendering a GCP workspace, for example, the Azure values should all remain as undefined.

Reply:
I defined these as variables just to make my test code easier to read/more concise, but I didn't intend them to become shared across multiple tests.

Reply:
As discussed in Slack, I'm happy to re-inline these if you want and then this file doesn't get changed at all. But for the time being at least your definition of them is perfect for what I needed, and does a good job of tracking the actual behavior defined in `useWorkspace`, so I'd like to keep them available in workspace-fixtures.ts if possible.